build.py

```diff
@@ -263,7 +268,7 @@ def psyko_module(psy_source):
 
     def psyko_partition(name, objects):
         generated_object = args.build_dir / (name + ".parto")
-        psyko("partition", "-o", generated_object, *objects)
+        psyko("partition", "--gendir", "GEN", "-o", generated_object, *objects)
```

Comment:
The resulting `GEN/` directory will pollute the source dir. This is fine for development, but should be removed.